aws/cloudformation/iam.yml.erb

```diff
@@ -306,7 +306,7 @@ roles = {
     GoogleIdsSSMParameter: 'DeveloperGoogleIDs',
     ManagedPolicyArns: [
       'arn:aws:iam::aws:policy/ReadOnlyAccess',
-      '!Ref DenySecretsPolicy'
+      '!Ref EngineerSecretsPolicy',
```

Comment:
This is a rename, and will be deployed as follows by Cloudformation:

- create new policy with new name
- update role to use new policy
- delete old, now unused, policy